### ISSUE TITLE
DateTime wallclocks may not be precise enough, consider timestamps offset from Unix epoch instead?

### ISSUE THREAD

**Issue by trgibeau** [microsoft/microsoft-performance-toolkit-sdk#100]
This will be a breaking change, but one that we're willing to take given that it will allow greater precision going forward.  We discussed two options:

1: The custom data processor converts all timestamps to be offsets from the unix epoch.
2: The custom data processor provides a wallclock before starting trace processing, and the runtime will do the conversion.

I'm not sure how #2 will work yet, we'll have to think that one through a bit more.

**Comment by trgibeau** [comment]
Not needed for 1.0. 
Will add a new interface or property in version 1.* that will allow for precision between data sources to be specified in a nano second precision.
